include/deal.II/lac/sparse_matrix.h

```diff
@@ -974,8 +986,14 @@ class SparseMatrix : public virtual Subscriptor
    * classes instead, since they are tailored better to a sparse matrix
    * structure.
    */
-  number operator () (const size_type i,
-                      const size_type j) const;
+  const number &operator () (const size_type i,
```

Comment:
technically, i don't need these changes anymore, but I guess it does not harm.

Reply:
Correct.